src/main/resources/application.yml

```diff
@@ -94,4 +99,8 @@ censusconfig:
 uprnconfig:
   dummyuprnprefix: 999
 
+pubsub:
+  publishtimeout: 30  # In seconds
+  aimstopic: aims-topic
```

Comment:
Worth naming this something like `new-address-topic` to clarify it's purpose?

Reply:
not a bad shout